homeassistant/components/hue/light.py

```diff
@@ -229,6 +229,11 @@ def __init__(self, light, request_bridge_update, bridge, is_group=False):
             self.gamut_typ = self.light.colorgamuttype
             self.gamut = self.light.colorgamut
             _LOGGER.debug("Color gamut of %s: %s", self.name, str(self.gamut))
+            if not color.check_valid_gamut(self.gamut):
+                err = "Color gamut of %s: %s, not valid, setting gamut to None"
+                _LOGGER.warning(err, self.name, str(self.gamut))
+                self.gamut_typ = 'None'
```

Comment:
This should probably become a constant since it's used in multiple places.

Reply:
I was referring to the string `'None'`

Reply:
Alright, that makes more sence.
I will make it a constant.